specs/olm.spec

```diff
@@ -48,4 +48,9 @@ Importance: Critical
 Uninstalls `openshift-pipelines` operator using olm
 Steps:
   * Validate Operator should be installed
-  * Uninstall Operator
\ No newline at end of file
+  * Uninstall Operator
+
+## Verify that Pipelines operator installs quickstarts: ID PIPELINES-09-TC06
+Tags: install, upgrade
+Steps:
+    * Validate quickstarts
```

Comment:
@VeereshAradhya I told Anton to create a new scenario but now I'm thinking if this singel step shouldn't go to installation scenario, somewhere after line 39. WDYT?